packages/toolkit/src/components/LatestChangesCard.tsx

```diff
@@ -0,0 +1,66 @@
+import { Icons } from "@instill-ai/design-system";
+import { useChangelogs } from "../lib/react-query-service/misc/useChangelogs";
+
+interface Changelog {
+  id: string;
+  createdAt: string;
+  updatedAt: string;
+  date: string | null;
+  title: string;
+  published: boolean;
+}
+
+
+const LatestChangesCard: React.FC = () => {
+  const { data: changelogs, isLoading, isError } = useChangelogs();
+
+  if (isLoading) {
+    return <div>Loading...</div>;
+  }
+
+  if (isError) {
+    return <div>Error fetching changelogs.</div>;
+  }
+
+  return (
+    <div className="mt-4 flex flex-col gap-y-2 rounded-sm border border-semantic-bg-line p-4">
+      <h2 className="mb-4 text-2xl font-bold">Latest Changes</h2>
+      {changelogs?.map((changelog: Changelog) => (
+        <div key={changelog.id}>
+          <button
+            type="button"
+            className="my-auto flex-auto rounded-sm bg-semantic-accent-bg p-2 capitalize text-semantic-accent-default product-button-button-2 hover:!underline"
+          >
+            {changelog.date && new Date(changelog.date).getTime() !== 0
+              ? new Date(changelog.date).toLocaleDateString("en-US", {
+                year: "numeric",
+                month: "long",
+                day: "numeric",
+              })
+              : new Date(changelog.updatedAt).toLocaleDateString("en-US", {
+                year: "numeric",
+                month: "long",
+                day: "numeric",
+              })}
+          </button>
+          <p>{changelog.title}</p>
+        </div>
+      ))}
```

Comment:
@thewbuk When utilize reqct-query. The correct way to make sure data is there is like

const changelogs = useChangelogs()

and you only need to check whether the query is success something like

```
if (changelogs.isSuccess){
  return changelogs.data.map() // <---- Don't need to do the optional chain anymore
}

```